DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/add_kernels.tcl

```diff
@@ -0,0 +1,486 @@
+package require -exact qsys 21.3
```

Comment:
Same remark: I hope that we can avoid shipping this file

Reply:
will move to regtest